app/components/ManageSpecies/SpecieInfo.js

```diff
@@ -1,21 +1,160 @@
-import React from 'react';
-import { View, StyleSheet, Image, SafeAreaView } from 'react-native';
-import { Header } from '../Common';
-import { Colors } from '_styles';
-import { add_image } from '../../assets';
+import React, { useState, useEffect } from 'react';
+import {
+  View,
+  StyleSheet,
+  SafeAreaView,
+  Text,
+  Image,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  TextInput,
+  Dimensions,
+  Platform,
+} from 'react-native';
+import { Header, Camera } from '../Common';
+import { Colors, Typography } from '_styles';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import RNFS from 'react-native-fs';
+import { UpdateSpeciesImage } from './../../utils/specieImageUpload';
+import { addAliases, addLocalImage } from './../../repositories/species';
+import { setSpecieAliases } from '../../actions/species';
+import { ScrollView } from 'react-native-gesture-handler';
 
 const SpecieInfo = ({ route }) => {
+  const [isCamera, setIsCamera] = useState(false);
+  const [aliases, setAliases] = useState('');
+  const [localImageUrl, setLocalImageUrl] = useState('');
   const specieName = route.params.SpecieName;
-  return (
-    <SafeAreaView style={styles.mainContainer}>
-      <View style={styles.container}>
-        <Header headingText={specieName} />
-        <View style={styles.scrollViewContainer}>
-          <Image source={add_image} style={styles.bgImage} />
-        </View>
+  const SpecieGuid = route.params.SpecieGuid;
+  const SpecieId = route.params.SpecieId;
+  let SpecieDescription = route.params.SpecieDescription;
+
+  useEffect(() => {
+    setAliases(route.params.SpecieAliases);
+    if (route.params.SpecieLocalImage) {
+      setLocalImageUrl(route.params.SpecieLocalImage);
+    }
+    if (route.params.SpecieImage && !route.params.SpecieLocalImage)
+      RNFS.downloadFile({
+        fromUrl: `https://bucketeer-894cef84-0684-47b5-a5e7-917b8655836a.s3.eu-west-1.amazonaws.com/development/media/cache/species/default/${route.params.SpecieImage}`,
```

Comment:
this should not be hardcoded.

Please use cdn url to fetch files like this

Reply:
Yes, I forgot to add CDN. I will add it in the current commit

Reply:
compare https://github.com/Plant-for-the-Planet-org/treemapper/blob/86665a75f748e6bdc2cf62a6f67452b82ca03f11/app/components/MainScreen/index.js#L245